### PR TITLE
i18n(ru): Add new `rendering-modes.mdx` translation

### PR DESCRIPTION
**Pull request by mearashadowfax**
#### Description

This PR adds new Russian translation of `src/content/docs/ru/basics/rendering-modes.mdx` page, addressing missing content flagged on the Translation Tracker.

### CODE REVIEW

File: src/content/docs/ru/basics/rendering-modes.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+title: "Режимы Рендеринга"
+i18nReady: true
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import RecipeLinks from '~/components/RecipeLinks.astro';
+
+Для отображения в Интернете код вашего проекта Astro должен быть **отрендерен** в HTML. 
+
+Страницы Astro, маршруты и конечные точки API могут быть либо [предварительно отрендерены на этапе сборки](#pre-rendered), либо [отрендерены сервером](#on-demand-rendered) при запросе пути. С [Островами Astro](/ru/concepts/islands/) вы также можете включить рендеринг на стороне клиента, если это необходимо.
```
Comment (yanthomasdev) [inline src/content/docs/ru/basics/rendering-modes.mdx:11]:
```suggestion
Страницы Astro, маршруты и конечные точки API могут быть либо [предварительно отрендерены на этапе сборки](#пререндеренные), либо [отрендерены сервером](#рендеринг-по-запросу) при запросе пути. С [Островами Astro](/ru/concepts/islands/) вы также можете включить рендеринг на стороне клиента, если это необходимо.
```

Comment (mearashadowfax) [reply src/content/docs/ru/basics/rendering-modes.mdx]: Thank you! Sorry, I still sometimes confuse the correct links.

File: src/content/docs/ru/basics/rendering-modes.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Для отображения в Интернете код вашего проекта Astro должен быть **отрендерен** в HTML. 
+
+Страницы Astro, маршруты и конечные точки API могут быть либо [предварительно отрендерены на этапе сборки](#пререндеренные), либо [отрендерены сервером](#рендеринг-по-запросу) при запросе пути. С [Островами Astro](/ru/concepts/islands/) вы также можете включить рендеринг на стороне клиента, если это необходимо.
+
+В Astro большая часть обработки происходит на сервере, а не в браузере. Это обычно делает ваш сайт или приложение быстрее при просмотре на менее мощных устройствах или при медленном интернет-соединении, чем рендеринг на стороне клиента. Серверный рендеринг HTML быстр, SEO-дружелюбен и доступен по умолчанию.
+
+## Режимы `output` сервера
+
+В конфигурации [`output`] (/ru/reference/configuration-reference/#output) вы можете настроить, как будут отображаться ваши страницы.
```
Comment (yanthomasdev) [inline src/content/docs/ru/basics/rendering-modes.mdx:17]:
```suggestion
В конфигурации [`output`](/ru/reference/configuration-reference/#output) вы можете настроить, как будут отображаться ваши страницы.
```

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks @mearashadowfax 🙌

**Pull request merged**